README.md

```diff
@@ -5,7 +5,7 @@ A prototype of the Design System created Rackspace Experience Design. This site
 
 Ensure ruby is installed on your system and run the following.
 
-```
+``` bash
 bundle install
-jekyll serve --watch
+bundle exec jekyll serve --watch --config _config.yml,_dev-config.yml
```

Comment:
I think we could, for sure. Right now, this should work for this upcoming surge as the infodev folks can run with this, and I'll get it added to a JIRA task.